cartridge-cli.lua

```diff
@@ -2322,6 +2343,21 @@ end
 
 local function app_pack(args)
     local name, release, version = detect_name_release_version(args.path, args.name, args.version)
+
+    -- collect general application info
```

Comment:
This is quite inconsistent with `app_create`...

I think that logic of this functions should be uniform

Reply:
`app_create` function is much more simple. It calls the only one function `instantiate_template` and pass in this function only one application parameter - name.

I think, that we shouldn't complicate `app_create` function.

Reply:
Ok